File: InschrijvingenApp/ClientApp/app/components/models/medisch.ts
export class Medisch {
    constructor() {
        this.huisArtsNaam = "";
        this.huisArtsTelefoon = "";
        this.ziekteIngreep = "";
        this.suikerZiekte = false;
        this.astma = false;
        this.hartkwaal = false;
        this.huidaandoening = false;
        this.epilepsie = false;
        this.allergieen = new Array<string>();
        this.andereAandoeningen = new Array<string>();
        this.aanpakKind = "";
        this.vaccinatieKindEnGezin = false;
        this.kanZwemmen = false;
        this.kanSporten = false;
        this.belemmeringenSport = "";
        this.geneesmiddelen = false;
        this.geneesmiddelenLijst = "";
        this.pijnstillersAllowed = false;
        this.fotosAllowed = false;
    }

    public huisArtsNaam: string;
    public huisArtsTelefoon: string;
    public ziekteIngreep: string;
    public suikerZiekte: boolean;
    public astma: boolean;
    public hartkwaal: boolean;
    public huidaandoening: boolean;
    public epilepsie: boolean;
    public allergieen: Array<string>;
    public andereAandoeningen: Array<string>;
    public aanpakKind: string;
    public vaccinatieKindEnGezin: boolean;
    public kanZwemmen: boolean;
    public kanSporten: boolean;
    public belemmeringenSport: string;
    public geneesmiddelen: boolean;
    public geneesmiddelenLijst: string;
    public pijnstillersAllowed: boolean;
    public fotosAllowed: boolean;
}

File: InschrijvingenApp/Entities/Medisch.cs
using InschrijvenPietieterken.Entities;

namespace InschrijvingPietieterken.Entities
{
    public class Medisch : EntityBase
    {
        public string HuisArtsNaam { get; set; }
        public string HuisArtsTelefoon { get; set; }
        public string ZiekteIngreep { get; set; }
        public bool Suikerziekte { get; set; }
        public bool Astma { get; set; }
        public bool Hartkwaal { get; set; }
        public bool Huidaandoening { get; set; }
        public bool Epilepsie { get; set; }
        public string Allergieen { get; set; } // ; seperated
        public string AndereAandodeningen { get; set; } // ; seperated
        public string AanpakKind { get; set; }
        public bool VaccinatieKindEnGezin { get; set; }
        public bool KanZwemmen { get; set; }
        public bool KanSporten { get; set; }
        public string BelemmeringenSport { get; set; }
        public bool Geneesmiddelen { get; set; }
        public string GeneesmiddelenLijst { get; set; }
        public bool PijnStillersAllowed { get; set; }
        public bool FotosAllowed { get; set; }
    }
}